platform/engineering/design-docs/separate-content-build.md

```diff
@@ -62,9 +62,9 @@
 The objective here is to define an approach to isolate the concerns of the
 content build from the front end application build and vice versa.
 
-This design document is intended for front end engineers and DevOps engineers on
-the Veteran-facing Services Platform (VSP) and other Veteran-Facing Services
-(VFS) teams. Most directly affected is the Content Management System (CMS) team.
+This design document is intended for front end and DevOps engineers on the
+Veteran-facing Services Platform (VSP) and Veteran-Facing Services (VFS) teams.
+The most directly affected group is the Content Management System (CMS) team.
```

Comment:
This statement seems at odds with not having a CMS team representative as approver, which I understood was because this has _minimal_ impact on CMS team. Can you clarify?

Reply:
Yeah, I think this is maybe a vestige of a past iteration. I'd be happy to put a reviewer on here from the CMS team, but the impact should be fairly minimal. :thinking: Maybe "most affected" meant "this change will require some work from them, which is more than any other VFS team." Not sure what I was talking about. :confused: 

At any rate, [I've talked with a few folks](https://dsva.slack.com/archives/CT4GZBM8F/p1585783218385800) from their team about this briefly, and invited them to review and comment on this PR. I outlined what I expect the impact to them will be.